buildenv/jenkins/variables/defaults.yml

```diff
@@ -271,7 +271,7 @@ ppc64_aix:
     8: 'aix-ppc64-normal-server-release'
     11: 'aix-ppc64-normal-server-release'
   node_labels:
-    build: 'ci.role.build && hw.arch.ppc64 && sw.os.aix.7_2'
+    build: 'ci.role.build && hw.arch.ppc64 && sw.os.aix.7_2 && !sw.tool.c++runtime.17_1'
```

Comment:
The title says "16.1", but this says "17_1": Which is correct?

Reply:
Updated it to require 16.1